### DISCUSSION TITLE
Grafana Source installation on Windows - Build error

### DISCUSSION THREAD

**Discussion by muhdadel** [grafana/grafana#38880] [Alerting]
Hello,

I have build the UI and shows no issues , but while I am trying to build the backend it shows

c:\Users\PC\grafana>go run build.go build
Version: 8.2.0, Linux Version: 8.2.0, Package Iteration: 1630928573pre
building binaries build
building grafana-server ./pkg/cmd/grafana-server
rm -r ./bin/windows-amd64/grafana-server.exe
rm -r ./bin/windows-amd64/grafana-server.exe.md5
go build -ldflags -w -X main.version= -X main.commit=unknown-dev -X main.buildstamp=1630928573 -X main.buildBranch=main -buildmode=exe -o ./bin/windows-amd64/grafana-server.exe ./pkg/cmd/grafana-server
# github.com/grafana/grafana/pkg/cmd/grafana-server/commands
pkg\cmd\grafana-server\commands\cli.go:154:12: undefined: server.Initialize
exit status 2
exit status 1

Iam using :

GO 1.17
Node.Js 5.10.16

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
